src/storage/service.ts

```diff
@@ -924,4 +925,29 @@ export class StorageService {
   async download (commp: string | CommP, options?: DownloadOptions): Promise<Uint8Array> {
     return await this.providerDownload(commp, options)
   }
+
+  /**
+   * Get proofset roots for a given proofset ID
+   * @param proofsetId - The ID of the proof set to fetch roots for
+   * @returns Array of root CIDs
+   */
+  async getProofsetRoots (proofsetId: string): Promise<string[]> {
```

Comment:
I don't really see the utility of this since it's not a complex operation. I'm asking @nijoe1 here what he's wanting from this: https://github.com/FilOzone/synapse-sdk/issues/102#issuecomment-2994830438

_Maybe_ it makes sense given that the schema is more complicated than this PR suggests, but it would be something more like `proofSetData.roots.map((r) => r.rootCid)`. Leave it for now (remove the Metadata call below as per my other comment) and implement it properly and we'll have a look at it.